index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <h1 id="title">User Experience Survey Form</h1>
+    <p id="description">Thank you for sharing your experience with the app.</p>
+    <form id="survey-form">
+      <label for="name" id="name-label">Name:</label><br />
+      <input
+        type="text"
+        placeholder="Enter your name"
+        id="name"
+        required
+      /><br />
+      <label for="email" id="email-label">E-mail:</label><br />
+      <input type="email" placeholder="E-mail" id="email" required /><br />
+      <label for="number" id="number-label">Enter Phone Number:</label><br />
+      <input
+        type="number"
+        placeholder="Enter Phone-number"
+        id="number"
+        min="1000000000"
+        max="9999999999"
+      /><br />
+      <label for="dropdown">Choose Your Gender:</label>
+      <select id="dropdown" name="dropdown">
+        <option value="male">Male</option>
+        <option value="female">Female</option>
+      </select>
+
+      <p>Please select your favorite Web language:</p>
+      <input type="radio" id="html" name="fav_language" value="HTML" />
+      <label for="html">HTML</label><br />
+      <input type="radio" id="css" name="fav_language" value="CSS" />
+      <label for="css">CSS</label><br />
+      <input
+        type="radio"
+        id="javascript"
+        name="fav_language"
+        value="JavaScript"
+      />
+      <label for="javascript">JavaScript</label>
+
+      <br />
+
+      <p>Please select your age:</p>
+      <input type="radio" id="age1" name="age" value="30" />
+      <label for="age1">0 - 30</label><br />
+      <input type="radio" id="age2" name="age" value="60" />
+      <label for="age2">31 - 60</label><br />
+      <input type="radio" id="age3" name="age" value="100" />
+      <label for="age3">61 - 100</label><br />
+
+      <br />
+
+      Student: <input type="checkbox" id="student" value="student" /> Teacher:
+      <input type="checkbox" id="teacher" value="teacher" /> Parent:
+      <input type="checkbox" id="parent" value="parent" />
```

Comment:
Proper use of labels with the elements above will benefit:
- Screen reader users (will read out loud the label, when the user is focused on the element)
- Users who have difficulty clicking on very small regions (such as checkboxes) - because when a user clicks the text within the <label> element, it toggles the input (this increases the hit area). 

Kindly add labels for the input elements.